plugins/catalog-backend/README.md

```diff
@@ -87,6 +87,25 @@ yarn start
 This will launch both frontend and backend in the same window, populated with
 some example entities.
 
+## Audit Events
+
+- **`ancestry-fetch`**: Tracks `GET` requests to the `/entities/by-name/:kind/:namespace/:name/ancestry` endpoint which return the ancestry of an entity.
+- **`batch-fetch`**: Tracks `POST` requests to the `/entities/by-refs` endpoint which return a batch of entities.
+- **`delete`**: Tracks `DELETE` requests to the `/entities/by-uid/:uid` endpoint which delete an entity. Note: this will not be a permanent deletion and the entity will be restored if the parent location is still present in the catalog.
+- **`facet-fetch`**: Tracks `GET` requests to the `/entity-facets` endpoint which return the facets of an entity.
+- **`fetch`**: Tracks `GET` requests to the `/entities` endpoint which returns a list of entities.
+- **`fetch-by-name`**: Tracks `GET` requests to the `/entities/by-name/:kind/:namespace/:name` endpoint which return an entity matching the specified entity ref.
+- **`fetch-by-uid`**: Tracks `GET` requests to the `/entities/by-uid/:uid` endpoint which return an entity matching the specified entity uid.
+- **`fetch-by-query`**: Tracks `GET` requests to the `/entities/by-query` endpoint which returns a list of entities matching the specified query.
+- **`refresh`**: Tracks `POST` requests to the `/entities/refresh` endpoint which schedules the specified entity to be refreshed.
+- **`validate`**: Tracks `POST` requests to the `/entities/validate` endpoint which validates the specified entity.
+- **`location-analyze`**: Tracks `POST` requests to the `/locations/analyze` endpoint which analyzes the specified location.
+- **`location-create`**: Tracks `POST` requests to the `/locations` endpoint which creates a location.
+- **`location-delete`**: Tracks `DELETE` requests to the `/locations/:id` endpoint which deletes a location as well as all child entities associated with it.
+- **`location-fetch`**: Tracks `GET` requests to the `/locations` endpoint which returns a list of locations.
+- **`location-fetch-by-entity-ref`**: Tracks `GET` requests to the `/locations/by-entity` endpoint which returns a list of locations associated with the specified entity ref.
+- **`location-fetch-by-id`**: Tracks `GET` requests to the `/locations/:id` endpoint which returns a location matching the specified location id.
```

Comment:
I am not well versed in audit logging patterns, but this all seems wasteful to me. How about making "classes" of operation instead, and registering the path it was made on as a label? You [already store `originalUrl`](https://github.com/backstage/backstage/pull/26372/files#diff-910e2f2d3a4d559b1d06dbc752061a3aaeb995b6e69a22a91b2e6fbd6c5c4b6bR329) but for patterned requests you probably want to add `route: req.route.path` as well.

```
auditor.createEvent({
  eventId: 'read-entity',
  meta: { entityRef: restBody.entityRef },
  request: req,
});
```

and then use the exact same `'read-entity'` no matter which of the differente entity-reading operations were made. If someone wants to distinguish between the various forms of read (which I think would be less likely), they'd filter on the request route label. That way when we add new read features like the new-ish `queryEntities`, it won't leave a hole in people's security auditing graphs and whatnot, but fall into the already well known bucket.